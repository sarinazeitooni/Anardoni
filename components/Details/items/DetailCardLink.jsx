import React from 'react';
import style from './scss/DetailCardLink.module.scss';
import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
import messages from "./messages/messages";
function DetailCardLink({item}){
    return(
        <React.Fragment>
            {
                item.links &&
                <div className={style['link-container']}>
                    {
                        item.links.map((link)=>{
                            return(
                                <div>
                                    <h3 className={style['link-title']}>{messages.linkTitle}</h3>
                                    <div className={style.link}><span> <ArrowBackIosIcon/> </span> {link}</div>
                                </div>
                            )
                        })
                    }
                </div>
            }
        </React.Fragment>
    )
};
export default DetailCardLink;