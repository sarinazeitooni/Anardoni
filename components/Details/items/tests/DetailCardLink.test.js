import React from 'react';
import { shallow } from 'enzyme';
import DetailCardLink from "../DetailCardLink";
describe("DetailCardLink component" , ()=>{
    const container = shallow(<DetailCardLink item={[]}/>);
    it("it should render a div",()=>{
        expect(container).toMatchSnapshot();
    });
    it("it should render a mobile-card-container" ,()=>{
        const links = shallow(<DetailCardLink item={["1","2"]}/>);
        expect(links.find('div.link-container')).toBeTruthy();
    });
});