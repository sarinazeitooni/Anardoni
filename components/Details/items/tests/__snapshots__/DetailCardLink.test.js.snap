// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailCardLink component it should render a div 1`] = `ShallowWrapper {}`;
